.github/workflows/precompiled.yaml

```diff
@@ -403,6 +418,29 @@ jobs:
         run: |
           echo "PRIVATE_REGISTRY=ghcr.io" >> $GITHUB_ENV
 
+      - name: Download base image artifact
+        if: ${{ ! (matrix.driver_branch == 535 && contains(matrix.kernel_version, 'ubuntu24.04')) }}
+        uses: actions/download-artifact@v4
+        with:
+          name: base-images-${{ matrix.driver_branch }}-${{ matrix.kernel_version }}
+          path: ./
+
+      - name: Publish base image
+        if: ${{ ! (matrix.driver_branch == 535 && contains(matrix.kernel_version, 'ubuntu24.04')) }}
+        run: |
+          LTS_KERNEL=$(echo "$input" | sed -E 's/^([0-9]+\.[0-9]+)\..*/\1/')
+          KERNEL_FLAVOR=$(echo "$input" | sed -E 's/^.*-[0-9]+-([a-zA-Z]+)-.*/\1/')
+          DIST=$(echo "$input" | sed -E 's/^.*-(ubuntu[0-9]+\.[0-9]+)$/\1/')
+          if [[ "${DIST}" == "ubuntu22.04" ]]; then
+            BASE_TARGET="jammy"
+          elif [[ "${DIST}" == "ubuntu24.04" ]]; then
+            BASE_TARGET="noble"
+          fi
```

Comment:
Out of scope for this PR -- I think I have mentioned this in the past, but if we always used `ubuntu22.04` or `ubuntu24.04` instead of the codenames, it would remove the need to perform this mapping in multiple places.